<launch>
    <arg name="file"/>
    <node pkg="pir_control" name="textfile_controller" type="controller2.py" output="screen">
        <param name="distance" value="0"/>
        <param name="file_name" value="$(arg file)"/>
    </node>
    <node pkg="pir_control" name="lidar" type="detect_obs.py" output="screen">
    </node>
    <node pkg="pir_control" name="mimic_spt_to_cmd" type="mimic_spt_to_cmd" output="screen">
    </node>
</launch>
